RFS/src/main/java/com/rfs/common/RestClient.java

```diff
@@ -34,7 +34,7 @@ public RestClient(ConnectionDetails connectionDetails) {
     }
 
     public Response get(String path, boolean quietLogging) throws Exception {
-        String urlString = connectionDetails.host + "/" + path;
+        String urlString = connectionDetails.url + "/" + path;
         
         URL url = new URL(urlString);
         HttpURLConnection conn = (HttpURLConnection) url.openConnection();
```

Comment:
If the plan is to deprecate this class, use the @deprecate annotation for it (before `class RestClient`) so that we know that the plan is to rally all of the code around one HTTP client solution.  As it is, it's pretty confusing with 2 different clients within one codebase/PR.

Reply:
I'm not sure whether we want to deprecate this class or not in the long run.  I would assume so, but honestly the only place we really need to use the greater abilities of the reactor-netty client is for reindexing; this is fine elsewhere.  For that reason, I left this in place for the time being.